TheFourFifths/consus-client
# [TFF-145] Checkin model

### Summary

Ability to check in a checked out model

### Checklist


### How to Review

1. Get the address of a model available for checkout (such as: m8y7nFLsT)
2. Go to the student page and check out a large number of that model
3. First, just click the "Check in" Button on the student panel. A single model should be checked in with accompanying toast.
4. You can use the arrows on the number input to change the number checked in at once.  Entering numbers also works, and any number greater than the number checked out or less than 1 will be changed to the max or 1 respectively.
5. Clicking check in all will unsurprisingly check in the rest of the model, regardless of how many there are.
6.  Play around with checking in/out models in different amounts.  Periodically check the model page to make sure the quantity is being updated appropriately.
7. Run tests and review code

### Links

- [TFF-145](https://msoese.atlassian.net/browse/TFF-145)
- TheFourFifths/consus/pull/72
